README.md

```diff
@@ -1,2 +1,4 @@
 # git-demo
 to demonstrate how to use git and GitHub
+
+Clara Li (https://github.com/statsmed-sheep)
```

Comment:
Could you please add a `-` so this becomes a list item?

Also, please make your name into a link that goes to that URL. The URL should not be visible.

Reply:
Second commit hopefully addressed both issues--let me know if I need to do anything else.